packages/circuit-ui/components/Dialog/Dialog.module.css

```diff
@@ -0,0 +1,139 @@
+.dialog {
+  position: fixed;
+  top: 0;
+  right: 0;
+  bottom: 0;
+  left: 0;
+  padding: 0;
+  margin: auto;
+  overflow: scroll;
+  pointer-events: none;
+  visibility: hidden;
+  border: none;
+  border-radius: var(--cui-border-radius-mega);
+  outline: none;
+  box-shadow: 0 3px 8px 0 rgb(0 0 0 / 20%);
+  opacity: 0;
+  transition:
+    opacity 0.3s ease-in-out,
+    visibility 0.3s ease-in-out;
+}
+
+.dialog.show {
+  pointer-events: auto;
+  visibility: visible;
+  opacity: 1;
+}
+
+@media (min-width: 480px) {
+  .dialog .content {
+    width: min-content;
+    min-width: 480px;
+    max-width: 90vw;
+    max-height: 90vw;
+  }
+}
+
+@media (max-width: 479px) {
+  .dialog {
+    top: unset;
+    bottom: 0;
+    left: 0;
+    width: 100%;
+    max-width: 100%;
+    visibility: visible;
+    border-bottom-right-radius: 0;
+    border-bottom-left-radius: 0;
+    opacity: 1;
+    transition: transform 0.3s ease-in-out;
+    transform: translateY(100%);
+  }
+
+  .dialog.show {
+    transform: translateY(0%);
+  }
+}
+
+/* Native Backdrop */
+.dialog::backdrop {
+  visibility: hidden;
+  background-color: var(--cui-bg-overlay);
+  opacity: 0;
+  transition:
+    opacity 0.3s ease-in-out,
+    visibility 0.3s ease-in-out;
+}
+
+.dialog.show::backdrop {
```

Comment:
I noticed that the backdrop isn't visible in the Chromatic snapshots, do you know why?